### ISSUE TITLE
Alignment pipe (bwa+samblaster) is broken in bcbio1.0 when lumpy is specified for SV calling.  

### ISSUE THREAD

**Issue by NeillGibson** [chapmanb/bcbio-nextgen#1660]
Hi, 

I am testing bcbio version 1.0a ( this commit to be specific: https://github.com/chapmanb/bcbio-nextgen/commit/fb4a04b63638ba25af80557acab7e3ca1e6edfbd ) with the following yaml:
```
# Template for whole genome Illumina variant calling with FreeBayes
# This is a GATK-free pipeline without post-alignment BAM pre-processing
# (recalibration and realignment)
---
details:
  - analysis: variant2
    genome_build: my_reference
    description:
    # to do multi-sample variant calling, assign samples the same metadata / batch
    metadata:
      batch: project_123
    algorithm:
      aligner: bwa
      mark_duplicates: true
      recalibrate: false
      realign: false
      variantcaller: freebayes
      svcaller: lumpy
      nomap_split_targets: 3000
      effects: false
      tools_off:
      - gemini
      # for targetted projects, set the region
      # variant_regions: /path/to/your.bed
resources:
    freebayes:
        options: [--genotype-qualities, --min-mapping-quality 20]
```

This results in this error for me with two different sets of fastq files that I could previously process with bcbio version 0.9.9. 

```
samblaster: Loaded 13 header sequence entries.
[fputs] Broken pipe
[E::sam_parse1] SEQ and QUAL are of different length
[W::sam_read1] parse error at line 19215
[bam_sort_core] truncated file. Aborting.
[W::sam_read1] parse error at line 6353
[bam_sort_core] truncated file. Aborting.
[W::sam_read1] parse error at line 321353
[main_samview] truncated file.
' returned non-zero exit status 1
```

Removing `svcaller: lumpy` from the yaml fixes this issue. I guess because of using another bwa pipe that does not use samblaster for the split to discordant + split bam.

**Comment by chapmanb** [comment]
Neill;
Sorry about the delay in responding to this. I've been working on getting a fix into a new bioconda package and the build system was down this week, so finally got a hopeful fix pushed.

We managed to get a reproducible segfault case for samblaster (GregoryFaust/samblaster#26) and Greg pushed a fixed version, 0.1.24. I hope that this will resolve this issue if you update in your installation:
```
bcbio_conda install -c bioconda samblaster
````
and then re-run the analysis. If this doesn't fix it for you please let us know and we can investigate further. Thanks again for reporting this.

**Comment by NeillGibson** [comment]
Hi Brad. No worries, thank you for solving this issue / having this issue solved. 
I can confirm that after updating samblaster the bwa+samblaster alignment pipe now works. 
The errors are gone and I see `.bam, -sr.bam and -disc.bam` files being created.
